tests/test_heavy_workload_timeout.py

```
@@ -0,0 +1,379 @@
+
+    @pytest.mark.integration
+    @pytest.mark.asyncio
+    async def test_cli_heavy_workload_timeout_resolution(self):
+        """Integration test for CLI heavy workload timeout resolution."""
+        
+        if not os.getenv('GOOGLE_API_KEY'):
+            pytest.skip("GOOGLE_API_KEY not available for CLI integration test")
+        
+        # Test the exact CLI command that currently times out
+        import subprocess
+        import tempfile
```

Ruff (0.12.2)

`tempfile` imported but unused

Remove unused import: `tempfile`

(F401)